src/core/call_every_handler.cpp

```diff
@@ -56,6 +56,7 @@ void CallEveryHandler::remove(const void* cookie)
     auto it = _entries.find(const_cast<void*>(cookie));
     if (it != _entries.end()) {
         _entries.erase(const_cast<void*>(cookie));
+        cookie = nullptr;
```

Comment:
But `cookie` is `const`?

Reply:
Nevermind, I guess the pointer is `const`, the cookie can be changed, apparently.